### DISCUSSION TITLE
Is it even worth applying to US positions anymore? And Canada postings seem dry.

### DISCUSSION THREAD

**Discussion by ShevinuM** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#141] [General]
I haven't had any interviews with them despite mass applying. I feel like the companies are no longer sponsoring J-1.

I don't see a lot of postings in Canada as well. Maybe it's too early but let's see.

**Comment by kanjurer** [comment]
I am not applying to US, it's a waste of time lol

**Reply by frederickberube0** [reply]
I had an interview at a san francisco company (I'm a canadian citizen with no work permit in the US) that I ended up failing but thats about it

**Comment by ahatanar** [comment]
Which company?

**Reply by frederickberube0** [reply]
Asana
